src/js/themes/base.d.ts

```diff
@@ -1210,6 +1210,10 @@ export interface ThemeType {
     background?: BackgroundType;
     clear?: {
       container?: BoxProps;
+      hover?: {
+        background?: BackgroundType;
```

Comment:
```suggestion
        background?: ColorType;
```
I think this should be ColorType since we aren't supporting all the types allowed with BackgroundType